docs/tutorials/heatmaps.ipynb

```diff
@@ -11,7 +11,7 @@
   },
```

Comment:
<p>I think <a href="https://github.com/quantumlib/Cirq/blob/master/docs/dev/notebooks.md#pre-release-notebooks" target="_blank">protocol</a> says to include a disclaimer at the top of the file for pre-release notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
done